lm_eval/tasks/ds1000.py

```diff
@@ -23,15 +23,15 @@
 
 
 def get_tasks():
```

Comment:
Small nit: can we make the function and class namings consistent with the other tasks ( use `create_all_tasks` `create_task` and `GeneralDS1000` like this [benchmark](https://github.com/bigcode-project/bigcode-evaluation-harness/blob/18eece8006f5b032e9a028c03416e5a74fb434cf/lm_eval/tasks/codexglue_code_to_text.py#L37))

Reply:
resolved [here](https://github.com/bigcode-project/bigcode-evaluation-harness/pull/39/commits/bf38f8c2923ed098ec52384323d74ce7730bba27)